rosdep/python.yaml

```
@@ -1811,6 +1811,16 @@ python-google-cloud-vision-pip:
   ubuntu:
     pip:
       packages: [google-cloud-vision]
+python-google-cloud-secret-manager:
```

Because this is a pip-only key, this should be:

```suggestion
python3-google-cloud-secret-manager-pip:
```

(also you'll have to move it in the correct place so it is alphabetically sorted)